tsconfig.json

```diff
@@ -1,33 +1,20 @@
 {
-    "compilerOptions": {
-        "allowJs": true,
-        "module": "commonjs",
-        "target": "es6",
-        "outDir": "out",
-        "lib": [
-            "dom",
-            "es6"
-        ],
-        "sourceMap": true,
-        "rootDir": "src",
-        /* Strict Type-Checking Option */
-        "strict": true,   /* enable all strict type-checking options */
-        /* Additional Checks */
-        "noUnusedLocals": true, /* Report errors on unused locals. */
-        "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
-        "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
-        "noUnusedParameters": false,  /* Report errors on unused parameters. */
-        "checkJs": true,
-        "noImplicitThis": true,
-        "strictNullChecks": true,
-        "esModuleInterop": true
-    },
-    "exclude": [
-        "node_modules",
-        ".vscode-test",
-        "out",
-        "resources/**",
-        "sariftypesgenerator",
-        "src/explorer"
-    ]
-}
\ No newline at end of file
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es2019",
+		"sourceMap": true,
+		
+		"experimentalDecorators": true,
```

Comment:
>"experimentalDecorators": true, [](start = 2, length = 31)

What does this do?

Reply:
Decorators are best explained [here](https://github.com/tc39/proposal-decorators). We mostly use decorators for for Mobx-related modifiers such as `@observable` `@observer` `@action` and `@computed`. However it is useful for other libraries also such as `@autobind`.